ddtrace/internal/packages.py

```diff
@@ -1,3 +1,4 @@
+from functools import lru_cache as cached
```

Comment:
Note that the default size of `lru_cache` is 128 so this change depends on #9743 otherwise the creating of various caches needs to set the `maxsize` argument